plugins/scaffolder-react/src/translation.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Copyright 2024 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { createTranslationRef } from '@backstage/core-plugin-api/alpha';
+
+/** @alpha */
+export const scaffolderReactTranslationRef = createTranslationRef({
+  id: 'scaffolder-react',
+  messages: {
+    passwordWidget: {
+      content:
+        'This widget is insecure. Please use [`ui:field: Secret`](https://backstage.io/docs/features/software-templates/writing-templates/#using-secrets) instead of `ui:widget: password`',
+    },
+    scaffolderPageContextMenu: {
+      moreLabel: 'more',
+      createLabel: 'Create',
+      editorLabel: 'Manage Templates',
+      actionsLabel: 'Installed Actions',
+      tasksLabel: 'Task List',
+    },
+    stepper: {
+      backButtonText: 'Back',
+      createButtonText: 'Create',
+      reviewButtonText: 'Review',
+      stepIndexLabel: 'Step {{index, number}}',
+      nextButtonText: 'Next',
+    },
+    templateCategoryPicker: {
+      title: 'Categories',
+    },
+    templateCard: {
+      noDescription: 'No description',
+      chooseButtonText: 'Choose',
+      noTemplatesFoundPrefix:
```

Comment:
Maybe lets have `templatesNotFoundText` and `templatesNotFoundLinkText` as the two fields, and always render the period instead.

Reply:
Period will be different in different languages, and the text order would be different in Chinese at least, we won't say `aaa. Learn more about [xxx].` but `aaa。访问 [xxx] 学习更多内容。`

And these are just a temporary solution for jsx, it would be replaced by #27193

Reply:
@benjdlambert News on this?

Reply:
I just commented in the attached PR https://github.com/backstage/backstage/pull/27193#issuecomment-2636494789

Reply:
I noticed #27193, but that's a new feature, should that block this PR?

Reply:
I think so yes, as we don't want to add translations and then immediately break them.  

The messages are the API after all, the `prefix` / `suffix` pattern is something we want to avoid.

Reply:
> Maybe we could leave the problematic translations as they are for now and just undo those ones to get the rest in and then come back to the those translations once we have a way to support these nested translations

Done. I've reverted it back.